.jenkins

```diff
@@ -51,19 +33,19 @@ pipeline {
         // example matches: 1.2.3, 1.2.3-dev
         tag pattern: '(0|[1-9]*)\\.(0|[1-9]*)\\.(0|[1-9]*)(-(\\S*))?$', comparator: "REGEXP"
       }
+      agent any
       environment {
         GITHUB_TOKEN = credentials('vio-bot-gh-token')
       }
       steps {
         script {
-          docker.image('vaporio/jenkins-agent-golang:latest').inside('-v /var/run/docker.sock:/var/run/docker.sock -v /usr/bin/docker:/usr/bin/docker -e GITHUB_TOKEN=$GITHUB_TOKEN -v $WORKSPACE:$PROJ_PATH --group-add docker') {
+          docker.image('vaporio/jenkins-agent-golang:master').inside('-v /var/run/docker.sock:/var/run/docker.sock -v /usr/bin/docker:/usr/bin/docker -e GITHUB_TOKEN=$GITHUB_TOKEN -v $WORKSPACE:$PROJ_PATH --group-add docker') {
```

Comment:
Note: this still runs on the jenkins master node executors due to `agent: any` declared above, so we run it in the image. this will probably fail now that i look at it.